mypy/server/deps.py

```diff
@@ -659,6 +659,24 @@ def visit_super_expr(self, e: SuperExpr) -> None:
                     break
 
     def visit_call_expr(self, e: CallExpr) -> None:
+        if isinstance(e.callee, RefExpr) and e.callee.fullname == 'builtins.isinstance':
```

Comment:
Is there a similar situation with `issubclass()`? It is more rare, but it looks like it can be easily added here.

Reply:
Yeah, we could support it as well, but I don't think that it's worth even the minimal effort. I looked at an internal Dropbox codebase, and `issubclass()` was only used 1/50 as frequently as `isinstance()`.